rundeckapp/grails-app/views/scheduledExecution/_edit.gsp

```diff
@@ -59,7 +60,22 @@
   </div>
 
   <div class="tab-pane active" id="tab_details" data-ko-bind="jobeditor">
-  <section class="section-space-lg">
+  <g:if test="${uiType=='next'}">
+    <g:set var="allowHTML"
+           value="${!(cfg.getString(config: "gui.job.description.disableHTML") in [true,'true'])}"/>
```

Comment:
hmm, we will need a way to load necessary config metadata for the GUI...possibly a metadata component can be used

Reply:
Anything in mind regarding how this component should look like?

Reply:
I'll make a design, this is ok for now